deploy/crds/sriovnetwork_v1_sriovnetworknodestate_crd.yaml

```diff
@@ -51,10 +53,6 @@ spec:
           type: object
         status:
           properties:
-            lastSyncError:
-              type: string
-            syncStatus:
-              type: string
```

Comment:
@pliurh each time I run `make gencode` there are these changes, is right? or there is something needed to be changed?